src/D2L.CodeStyle.Analyzers/D2L.CodeStyle.Analyzers.nuspec

```diff
@@ -0,0 +1,29 @@
+﻿<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>$id$</id>
+    <version>$version$</version>
+    <title>$title$</title>
+    <authors>$author$</authors>
+    <owners>$author$</owners>
+
+    <projectUrl>https://github.com/Brightspace/D2L.CodeStyle</projectUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>$description$</description>
+    <copyright>$copyright$</copyright>
+    
+    <frameworkAssemblies>
+      <frameworkAssembly assemblyName="System" targetFramework="" />
+    </frameworkAssemblies>
+
+    <dependencies>
+      <dependency id="D2L.CodeStyle.Analysis" version="$version$" />

```

Comment:
The idea is that Analysis has the core analysis stuff (utilities, visitors, etc.), and that Analyzers is actually just Roslyn analyzers that wrap that stuff. That makes it very clear that "these have Roslyn analyzers" and "this has stuff that all roslyn analyzers use".

If the distinction ends up being bad in practice, I can squash the projects into just Analyzers later on.